macros/SpecName.ejs

```diff
@@ -408,8 +408,8 @@ var specList = {
         url  : 'https://drafts.csswg.org/cssom-view/'
     },
     'Custom Elements':{
-        name : 'Custom Elements',
-        url  : 'https://w3c.github.io/webcomponents/spec/custom/'
+        name : 'HTML (Custom Elements)',
+        url  : 'https://html.spec.whatwg.org/multipage/custom-elements.html#custom-elements'
```

Comment:
I removed all calls to {{SpecName('Custom Elements')}}. We shouldn't use this anymore. New pages will call {{SpecName('HTML')}}